src/ingestion/utils.ts

```diff
@@ -166,7 +166,15 @@ export function timeoutGuard(message: string, timeout = defaultConfig.TASK_TIMEO
 }
 
 const campaignParams = new Set(['utm_source', 'utm_medium', 'utm_campaign', 'utm_content', 'utm_term'])
-const initialParams = new Set(['$browser', '$browser_version', '$current_url', '$os', '$referring_domain', '$referrer'])
+const initialParams = new Set([
+    '$browser',
+    '$browser_version',
+    '$device_type',
```

Comment:
IMO we should add `$device` in here too then.

Reply:
The problem is that `$device` is just not set when you're not on a mobile device. Thus we the var would actually mean `$first_used_mobile_device`... and cause issues with accurate statistics.

Reply:
Hm, right, quirky prop. OK then